wsd/ClientSession.cpp

```diff
@@ -2020,7 +2020,7 @@ void ClientSession::postProcessCopyPayload(const std::shared_ptr<Message>& paylo
                 // clipboardcontent: content.application/x-openoffice-svxb;windows_formatname="SVXB (StarView Bitmap/Animation)"
                 // Do not issue this in those cases. (We should also cap the data we dump here.)
                 LOG_DBG("Missing <body> in textselectioncontent/clipboardcontent payload:\n"
-                        << Util::dumpHex(data));
+                        << [&](auto& log) { Util::dumpHex(log, data); });
```

Comment:
With configure --enable-coplugin there is a warning here of:

wsd/ClientSession.cpp:2023:64: warning: function parameter captured by reference, capture by value instead [coplugin:capture]

Reply:
Copying the data will surely be unhelpful, performance-wise. The plugin seems over-protective. Here, we don't store or forward the data.